lib/rules/no-restricted-imports.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @fileoverview Restrict usage of specified node imports.
+ * @author Guy Ellis
+ */
```

Comment:
Please add copyright and license line. See http://eslint.org/docs/developer-guide/code-conventions.html#file-format